src/path.jl

```diff
@@ -238,9 +238,8 @@ julia> p"foo" / "bar" / "baz"
 p"foo/bar/baz"
 ```
 """
-function Base.:(/)(root::AbstractPath, pieces::Union{AbstractPath, AbstractString}...)
-    join(root, pieces...)
-end
+/(root::AbstractPath, pieces::Union{AbstractPath, AbstractString}...) = joinpath(root, pieces...)
```

Comment:
Why doesn't this use `join`?
```suggestion
/(root::AbstractPath, pieces::Union{AbstractPath, AbstractString}...) = join(root, pieces...)
```

Reply:
Was considering dropping `join` all together.